requirements.txt

```diff
@@ -1,6 +1,6 @@
 ddt
 django<1.9,>=1.8
 git+https://github.com/edx/XBlock.git#egg=XBlock
-git+https://github.com/edx/xblock-utils.git@v1.0.0#egg=xblock-utils==v1.0.0
+git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==v1.1.1
```

Comment:
What is the impact of this re: ginkgo vs hawthorn, etc.?

Reply:
Thanks @stvstnfrd. I'll test this one, adding it to the pull request TODO list.

Reply:
Thanks for testing @OmarIthawi . I looked at the relevant commits and this does look like it should be backwards compatible.